### PR TITLE
Add type stubs for more functions

### PR DESCRIPTION
**Pull request by ryanrightmer**
A few more functions with types derived from the RDKit source and documentation.

### CODE REVIEW

File: rdkit-stubs/Chem/rdChemReactions.pyi [hunk 1]
```diff
@@ -92,7 +94,9 @@ class ChemicalReaction:
     @classmethod
     def RunReactantInPlace(cls, *args, **kwargs) -> Any: ...
     @classmethod
-    def RunReactants(cls, RDKit, boost) -> Any: ...
+    def RunReactants(
+        cls, reactants: Sequence[Mol], max_products=1000
```
Comment (bddap) [inline rdkit-stubs/Chem/rdChemReactions.pyi]: Is this really a class method?

Comment (bddap) [reply rdkit-stubs/Chem/rdChemReactions.pyi]: Same question applies to some other instances I see of this pattern in this pr.

**Review by bddap — APPROVED** [review]
1. Those instances of `@classmethod` look dubious.
2. There is a failing test do you understand what is going wrong?

File: rdkit-stubs/Chem/rdChemReactions.pyi (hunk 1 above)
Comment (ryanrightmer) [reply rdkit-stubs/Chem/rdChemReactions.pyi]: I neglected to remove the classmethods, will do now.

**Comment by ryanrightmer** [comment]
@bddap it appears that test has been failing for a few weeks now

**Review by bddap — APPROVED** [review]

**Comment by bddap** [comment]
Sounds like ci tests are broken. Running tests locally against main does not fail. (`./test/test/bash`).

This branch seems to introduce an error in tests that is obscured by the false errors in ci. I will push a pr to fix it.

**Comment by ryanrightmer** [comment]
@bddap thank you. I'll merge now and we can commence with a new release.

**Pull request merged**